### ISSUE TITLE
Google Play Games now supports friends :)

### ISSUE THREAD

**Issue by RuiBEAT** [playgameservices/play-games-plugin-for-unity#2872]
Hi,

First, let me thank you for this awesome plugin! :)

Now, the question: Since the last Google Play Games update (April 6th), the Google Play Games app now supports add users as friends. Is this feature planned to be added to this plugin in a short term?

Thank you in advance!

**Comment by olehkuznetsov** [comment]
Yes, this feature will come available to game developers soon. Stay tuned.
